_dashboards/search-telemetry.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Search telemetry
+nav_order: 30
+---
+
+
+# About search telemetry
+
+You can use search telemetry to analyze performance for success or failed search requests in OpenSearch Dashboards. OpenSearch stores telemetry data in the `.kibana_1` index.
+
+Because there are thousands of concurrent search request from OpenSearch Dashboards, the large traffic causes significant load in an OpenSearch cluster.
```

Comment:
"search request" should be plural here. Also, should we rewrite the second half to "the large traffic *can* cause significant load"?